components/wordPrompt.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.nextWords.length != 0) {
+        return {
+          ...lastWords,
+          currentWord: lastWords.nextWords.shift(),
+        };
+      } else {
+        return {
+          ...lastWords,
+          currentWord: "Out of words!",
+        };
+      }
+    });
+  };
+  const [words, setWords] = useState({
+    currentWord: "Start",
+    nextWords: [],
+  });
+  const fetchWords = () => {
+    fetch("https://random-word-api.herokuapp.com/word?number=10", {
+      method: "GET",
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setWords((lastWords) => {
+          lastWords.nextWords.push(...data);
```

Comment:
Be careful of the order of pushing/popping words here. Since we are popping words from the end of the array in `changeWordHandler`, shouldn't we add words to the front of the array? Something like
```
return [ ...data, ...lastWords]
```
Assuming that you followed my above comment and `lastWords` is just the `nextWords` array in the state